src/prefect/orion/api/deployments.py

```diff
@@ -136,6 +136,7 @@ async def read_deployments(
     flow_runs: schemas.filters.FlowRunFilter = None,
     task_runs: schemas.filters.TaskRunFilter = None,
     deployments: schemas.filters.DeploymentFilter = None,
+    sort: schemas.sorting.DeploymentSort = Body(schemas.sorting.DeploymentSort.NAME_ASC),
```

Comment:
Does sorting by default cost us query performance?

Reply:
cc @chrisguidry ?

Reply:
I do not know. I just tried to follow what flows is doing currently.

Reply:
👍 We'll want to check for indices on these columns before merging as well.

Reply:
Sorting by default should be fine performance-wise. We should add an index on `deployment.created` though. Happy to commit to this branch if you'd like

Reply:
Please do! Thanks @zangell44

Reply:
pushed!